src/unstructured_client/_hooks/custom/request_utils.py

```diff
@@ -69,10 +70,34 @@ async def call_api_async(
     )
 
     async with limiter:
-        response = await client.send(new_request)
+        response = await send_request_async_with_retries(client, new_request)
         return response
 
 
+async def send_request_async_with_retries(client: httpx.AsyncClient, request: httpx.Request):
+    retry_config = utils.RetryConfig(
+        "backoff",
+        utils.BackoffStrategy(2000, 60000, 1.5, 900000),
+        True
+    )
+
+    retryable_codes = [
```

Comment:
I guess it's a hard to eliminate tradeoff to hardcode here - the SDK uses `openapi.json` to read these values.

Reply:
Yes, I would love to be able to create the new httpx requests and just send them back into `general.partition`, to reuse the retry config and any other client settings. We'll need speakeasy to add a reference to the sdk in the hook context, but then we'll be able to really clean up `split_pdf_hook.py`. 

I added a comment to make it clear why this is hardcoded.